Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -335,8 +336,18 @@ class SettingsTableViewController: UITableViewController {
     override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let headerView = tableView.dequeueReusableHeaderFooterViewWithIdentifier(SectionHeaderIdentifier) as! SettingsTableSectionHeaderFooterView
         let sectionSetting = settings[section]
+
         if let sectionTitle = sectionSetting.title?.string {
             headerView.titleLabel.text = sectionTitle
+            if sectionTitle == Bug1209692 {
```

Comment:
I don't think we need this logic here since we override `viewForHeaderInSection` in `AppSettingsTableViewController`. Can we move this logic into that method where we dequeue the header? It would remove this string check since we already know when we dequeue the header.